cirq-google/cirq_google/engine/engine_program.py

```diff
@@ -543,8 +370,8 @@ async def get_circuit_async(self, program_num: Optional[int] = None) -> cirq.Cir
         supported if the program was created with the V2 protos.
 
         Args:
-            program_num: if this is a batch program, the index of the circuit in
-                the batch.  This argument is zero-indexed. Negative values
+            program_num:
```

Comment:
Can we remove/deprecate this argument? I think it only has meaning in the context of a batch program.